docs/how-to/run-ipfs-inside-docker.md

```diff
@@ -71,6 +71,33 @@ When starting a container running ipfs for the first time with an empty data dir
 docker run -d --name ipfs_host -e IPFS_PROFILE=server -v $ipfs_staging:/export -v $ipfs_data:/data/ipfs -p 4001:4001 -p 4001:4001/udp -p 127.0.0.1:8080:8080 -p 127.0.0.1:5001:5001 ipfs/go-ipfs:latest
 ```
 
+## Customizing your node
+Custom initialization code can be run by mounting scripts in the `/container-init.d` directory in the container. These are executed sequentially and in lexicographic order, after `ipfs init` is run and the swarm keys are copied (if the IPFS repo needs initialization), and before the IPFS daemon is started.
+
+Since this runs each time the container is started, you should check if your initialization code should be idempotent, particularly if you're persisting state outside of the container (e.g. using mounted directories).
+
+For example, this sets a custom bootstrap node before the daemon starts:
+
+```
+#!/bin/sh
+set -ex
+ipfs bootstrap rm all
+ipfs bootstrap add "/ip4/$PRIVATE_PEER_IP_ADDR/tcp/4001/ipfs/$PRIVATE_PEER_ID"
+```
+
+This shows how to mount the file into the `container-init.d` directory when running the container:
+```
+docker run -d --name ipfs \
+  -e PRIVATE_PEER_ID=... \
+  -e PRIVATE_PEER_IP_ADDR=... \
+  -v ./001-test.sh:/container-init.d/001-test.sh \
+  -p 4001:4001 \
+  -p 127.0.0.1:8080:8080 \
+  -p 127.0.0.1:5001:5001 \
+  ipfs/go-ipfs
+```
+
```

Comment:
Not sure if this is the right place, but I'd like to point people at examples repo created by @thattommyhall 

```suggestion

::: tip Use in custom images

See `gateway` example at https://github.com/ipfs-shipyard/go-ipfs-docker-examples

:::
```

Reply:
```suggestion
::: tip Use in custom images
See `gateway` example at[github.com/ipfs-shipyard/go-ipfs-docker-examples](https://github.com/ipfs-shipyard/go-ipfs-docker-examples)
:::
```

Sure, but let's format it like this.